data/alpha/field/local-authority.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: curie
+field: local-authority
+phase: alpha
+register: ''
+text: 'A local authority'
```

Comment:
update to 
'A local authority in England, Scotland and Northern Ireland or a principal local authority in Wales.'

Reply:
Also rename to `local-authorities` and have `cardinality: "n"`

Reply:
Also rename to `local-authorities` and have `cardinality: 'n'`

Reply:
ammended